html/includes/graphs/application/ups-nut_remaining.inc.php

```diff
@@ -13,10 +13,14 @@
 * You should have received a copy of the GNU General Public License
 * along with this program.  If not, see <http://www.gnu.org/licenses/>.
 *
+* @version    1.1
```

Comment:
I don't think we need version numbers. It could just get confusing.

We have commit hashes in git which can be used to determine file versions if you need to.